components/net_traits/lib.rs

```diff
@@ -504,6 +506,14 @@ impl ResourceFetchTiming {
     // TODO currently this is being set with precise time ns when it should be time since
     // time origin (as described in Performance::now)
     pub fn set_attribute(&mut self, attribute: ResourceAttribute) {
+        let should_attribute_always_be_updated = match attribute {
+            ResourceAttribute::FetchStart => true,
+            ResourceAttribute::ResponseEnd => true,
```

Comment:
I think we can move this into same arm with `FetchStart` because they have the same result 👀?

```rust
match attribute {
    ResourceAttribute::FetchStart | ResourceAttribute::ResponseEnd => true,
    _ => false,
}
```

Reply:
Thanks! I didn't know about that syntax.